### PR TITLE
Small Refactor: move call composite side pane state handling to its own hook

### PR DESCRIPTION
**Pull request by JamesBurnside**
# What
Instead of defining useState and all the handlers in the (already big) Call Arrangement file, move to its own useState-style hook file

# Why
Move business logic out of the view code. If we have more complex side pane content this file will grow even more large

# How Tested
Storybook, people pane showed and closed as expected in 1-n call:
image

### CODE REVIEW

File: packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx [hunk 1]
```diff
@@ -16,17 +16,14 @@ import {
...
 import { AvatarPersonaDataCallback } from '../../common/AvatarPersona';
 import { containerDivStyles } from '../../common/ContainerRectProps';
 /* @conditional-compile-remove(one-to-n-calling) */
 import { useAdapter } from '../adapter/CallAdapterProvider';
 import { CallControls, CallControlsProps } from '../components/CallControls';
-/* @conditional-compile-remove(one-to-n-calling) */
-import { useSelector } from '../hooks/useSelector';
-/* @conditional-compile-remove(one-to-n-calling) */
-import { callStatusSelector } from '../selectors/callStatusSelector';
+import { useSidePaneState } from '../hooks/useSidePaneState';
```
Comment (edwardlee-msft) [inline packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx:26]: Should this be conditionally compiled as well?

Comment (JamesBurnside) [reply packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx:27]: indeed! thanks

Comment (edwardlee-msft) [reply packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx:27]: This should be conditionally compiled for one-to-n-calling

Comment (JamesBurnside) [reply packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx:27]: 🤦 - updated!

Comment (JamesBurnside) [reply packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx:27]: (my secret plan next PR is to remove the fact we have two people panes and make callwithchat hook into this so would remove the cc line, but also involves doing controversial things like creating an imperative handle to the call composite to control the pane from the callwithchat composite 👀)

**Review by edwardlee-msft — APPROVED** [review]
Love the extraction!!

**Review by dmceachernmsft — APPROVED** [review]

**Review by anjulgarg — APPROVED** [review]

**Pull request merged**